generate_protos.log

```diff
@@ -12,11 +10,11 @@ Date:   Tue Jan 16 13:30:08 2018 -0800
     
     Signed-off-by: Brian Pane <bpane@pinterest.com>
 ~/go/src/github.com/envoyproxy/go-control-plane
-Generating gogofast protos api ...
-Generating gogofast protos api/auth ...
-Generating gogofast protos api/filter ...
-Generating gogofast protos api/filter/accesslog ...
-Generating gogofast protos api/filter/http ...
-Generating gogofast protos api/filter/network ...
+Generating protos api ...
+Generating protos api/auth ...
+Generating protos api/filter ...
+Generating protos api/filter/accesslog ...
+Generating protos api/filter/http ...
+Generating protos api/filter/network ...
```

Comment:
Why do we commit this log file?

Reply:
To record the version of data-plane-api and versions of the tools.